src/SonarScanner.MSBuild.Shim/ScannerEngineInputGenerator.cs

```diff
@@ -415,4 +419,23 @@ string KeySuffix()
             }
         }
     }
+
+    private void AddCmdLineArgs(ScannerEngineInput engineInput)
+    {
+        var properties = cmdLineArgs.GetAllProperties().ToList();
+        properties.AddRange(SensitiveFileProperties());
```

Comment:
This does not belong to `AddCmdLineArgs`. It should be in something like `AddSensitiveProperties`
```suggestion
```